test/language/module-code/comment-single-line-html-close-comment-before-function.js

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2017 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: >
+    HTML-like comments are not available in module code
+    (SingleLineHTMLCloseComment)
+esid: sec-html-like-comments
+es6id: B1.3
+negative:
+  phase: early
+  type: SyntaxError
+flags: [module]
+---*/
+
+-->
+function f(){}
```

Comment:
What is it about these two lines that differentiates it from any `-->\n?` where `?` is anything else?